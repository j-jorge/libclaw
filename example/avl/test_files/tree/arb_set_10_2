 77 73 75 30 4 43 11 83 38 4 20