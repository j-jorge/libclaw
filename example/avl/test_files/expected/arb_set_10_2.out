Tree is empty
File loading...
size : 10
Min = 4
Max = 83
Tree's not empty
Found : 4 11 20 30 38 43 73 75 77 83
Scan with an iterator : 4 11 20 30 38 43 73 75 77 83
Scan bis, iterator affectation :4: 4 11 20 30 38 43 73 75 77 83
11: 11 20 30 38 43 73 75 77 83
20: 20 30 38 43 73 75 77 83
30: 30 38 43 73 75 77 83
38: 38 43 73 75 77 83
43: 43 73 75 77 83
73: 73 75 77 83
75: 75 77 83
77: 77 83
83: 83
Deletions
Copy and scan
source size : 0
copy size : 0
Tree's copy is empty

Reverse scan

Assignment
Clear
Tree's is empty
